docusaurus/docs-internals/editor-controls/data-controls/toggle-button.md

```diff
@@ -0,0 +1,275 @@
+---
+sidebar_position: 1
```

Comment:
`sidebar_position: 2 `

but theoretically you can remove this because all gyus will write second position)

and this position we will add at the final